### PR TITLE
Cleanups

### PR DESCRIPTION
**Pull request by sbird**
Cleanups of the remaining random compile-time options, delete a couple of obsolete files, and a couple of fixes for compilation on stampede.

### CODE REVIEW

File: depends/Makefile [hunk 1]
```diff
@@ -17,7 +18,7 @@ lib/libbigfile-mpi.a: bigfile/src/bigfile-mpi.c
 	cd bigfile/src; \
 	make install PREFIX=$(PWD) CC="$(MPICC)" MPICC="$(MPICC)" CFLAGS="$(OPTIMIZE)"
 
-lib/libpfft_omp.a:
+pfft:
```
Comment (rainwoodman) [inline depends/Makefile]: Ah. Good catch.

File: libgadget/cooling_rates.c [hunk 2]
```diff
@@ -174,12 +184,12 @@ load_treecool(const char * TreeCoolFile)
                 continue;
             Gamma_log1z[i] = atof(retval);
             /*Get the rest*/
-            Gamma_HI.ydata[i] = log10(atof(strtok(NULL, " \t")));
-            Gamma_HeI.ydata[i] = log10(atof(strtok(NULL, " \t")));
-            Gamma_HeII.ydata[i] = log10(atof(strtok(NULL, " \t")));
-            Eps_HI.ydata[i] = log10(atof(strtok(NULL, " \t")));
-            Eps_HeI.ydata[i] = log10(atof(strtok(NULL, " \t")));
-            Eps_HeII.ydata[i] = log10(atof(strtok(NULL, " \t")));
+            Gamma_HI.ydata[i]   = load_tree_value();
```
Comment (rainwoodman) [inline libgadget/cooling_rates.c]: Better to use a locally defined macro than hiding strtok, which is a weird function that uses a hidden state. We may forget load_tree_value also has that strange behaviour sooner or later.

Comment (sbird) [reply libgadget/cooling_rates.c]: Even better to not use the weird strtok function: the last commit has strtok_r

File: depends/Makefile (hunk 1 above)
Comment (sbird) [reply depends/Makefile]: Thanks! It was really Yueying who found it. Unfortunately I needed an extra commit to tell Make not to keep trying to create this file...

**Pull request merged**